.gitignore

```diff
@@ -21,6 +21,7 @@
 
```

Comment:
I would suggest adding Unity-specific lines, like [here](https://github.com/github/gitignore/blob/master/Unity.gitignore), since Unity projects tend to get messy overtime.